src/icon/mixins.scss

```diff
@@ -4,81 +4,72 @@
 
 @mixin iui-icons-default {
   @include iui-icons-color();
+  display: inline-flex;
   width: $iui-icons-default;
   height: $iui-icons-default;
 }
 
 @mixin iui-icons-small {
   @include iui-icons-color();
+  display: inline-flex;
   width: $iui-icons-small;
   height: $iui-icons-small;
 }
 
 @mixin iui-icons-large {
   @include iui-icons-color();
+  display: inline-flex;
   width: $iui-icons-large;
   height: $iui-icons-large;
 }
 
 @mixin iui-icons-xl {
   @include iui-icons-color();
+  display: inline-flex;
   width: $iui-icons-xl;
   height: $iui-icons-xl;
 }
 
 @mixin iui-icons-xxl {
   @include iui-icons-color();
+  display: inline-flex;
   width: $iui-icons-xxl;
   height: $iui-icons-xxl;
 }
 
 @mixin iui-icons-3xl {
   @include iui-icons-color();
+  display: inline-flex;
   width: $iui-icons-3xl;
   height: $iui-icons-3xl;
 }
 
 @mixin iui-icons-color {
-  &,
-  path {
```

Comment:
So now no need to apply to path?

Reply:
From what I understand, this `path` hack was needed because some of our svgs had inline `fill`. But we fixed that a while ago.

Here, applying the fill to path in this mixin was causing some issues where the path needed to be overridden if we want to change fill where it's used.